src/scores/categorical/contingency_impl.py

```diff
@@ -1131,10 +1213,15 @@ def make_contingency_manager(
 
 class ThresholdEventOperator(EventOperator):
     """
-    Given a forecast and and an observation, consider an event defined by
-    particular variables meeting a threshold condition (e.g. rainfall above 1mm).
+    See https://scores.readthedocs.io/en/stable/tutorials/Binary_Contingency_Scores.html for
+    a detailed walkthrough showing the use in practice.
+
+    A ThresholdEventOperator is used to produce a :py:class:`BinaryContingencyManager` from
+    forecast and observed data. It considers an event to be defined when the forecast and
+    observed variables meet a particular threshold condition (e.g. rainfall above 1mm).
 
-    This class abstracts that concept for any event definition.
+    The class may be used for any variable, for any threshold, and for any comparison
+    operator (e.g. greater-than, less-than, greater-than-or-equal-to, ... )
```

Comment:
Would it be better for this sentence to end with "etc." or "so forth" or something like that, rather than with "..."?